callisto/delivery/models.py

```diff
@@ -15,6 +16,35 @@
 from django.utils.html import strip_tags
 
 
+def _encrypt_report(salt, key, report_text):
+    stretched_key = pbkdf2(key, salt, settings.KEY_ITERATIONS, digest=hashlib.sha256)
+    box = nacl.secret.SecretBox(stretched_key)
+    message = report_text.encode('utf-8')
+    nonce = nacl.utils.random(nacl.secret.SecretBox.NONCE_SIZE)
+    return box.encrypt(message, nonce)
+
+
+def _decrypt_report(salt, key, encrypted):
+    stretched_key = pbkdf2(key, salt, settings.KEY_ITERATIONS, digest=hashlib.sha256)
+    box = nacl.secret.SecretBox(stretched_key)
+    decrypted = box.decrypt(bytes(encrypted))
+    return decrypted.decode('utf-8')
+
+
+def _pepper(encrypted_report):
```

Comment:
@swenson I read a bunch of conflicting internet advice on peppers, which seemed mainly to revolve around the lack of an agreed-upon definition of the term. I ended up going with the final advice [here](http://stackoverflow.com/a/16896216/1308611), cause it seems to give most of the benefit of a pepper (adding protection in case of DB but not server breach) while offering a simple path for upgrading existing entries & future key rotation. Let me know what you think.